src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -138,10 +138,6 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
 
                     $attributeMetadata->setSerializedName($annotation->getSerializedName());
                 } elseif ($annotation instanceof Ignore) {
-                    if (!$accessorOrMutator) {
-                        throw new MappingException(sprintf('Ignore on "%s::%s()" cannot be added. Ignore can only be added on methods beginning with "get", "is", "has" or "set".', $className, $method->name));
-                    }
-
                     $attributeMetadata->setIgnore(true);
```

Comment:
maybe silently ignoring the attribute on non-accessor and non-mutator methods can help:

```php
if ($accessorOrMutator) {
    $attributeMetadata->setIgnore(true);
}
```

Reply:
updated to your suggestion

Reply:
@Jeroeny could you please check if this works with #45016 in mind?